### PR TITLE
Week 7 18 22 dependency updates

### PR DESCRIPTION
**Pull request by rgoshen**
This round of updates includes:

- Publishing a new ECR image 2.1.0
- Update Terraform to 1.2.5
- Passing [CI Workflow](https://app.circleci.com/pipelines/github/flexion/ef-cms/36544/workflows/1f68fc2d-ba1a-456a-86b2-3cca28a8b09b) with a migration

**Major package updates:**

- luxon
- stylelint-config-standard-scss

**Minor package updates:**

- aws-sdk
- broadcast-channel
- esbuild-css-modules-plugin
- esbuild-sass-plugin
- eslint
- eslint-plugin-jest

**Patch package updates:**

- core-js
- @babel/cli
- @babel/core
- @babel/eslint-parser
- @babel/preset-env
- @babel/register
- artillery
- babel-jest
- cypress
- jest
- jest-environment-jsdom
- jest-environment-node

**Caveats added:**

- None

**Caveats resolved:**

- broadcast-channel

**Misc:**

- removed @babel/core from team-process.md caveat section since it had already been updated from the caveat version mentioned in the document.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,6 +65,7 @@
     "riot-route": "^3.1.4",
     "s3-files": "^2.0.3",
     "sanitize-filename": "^1.6.3",
+    "stylelint-config-standard-scss": "^5.0.0",
```
Comment (rosiehsmith) [inline package.json]: is this a dev dependency?

**Review by rosiehsmith — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (rgoshen) [reply package.json]: I am not sure how that happened.  It was a dev dependency before.  I will move it back to dev dependency.

Comment (rgoshen) [reply package.json]: It has been moved back to dev dependency.

**Review by TomElliottFlexion — APPROVED** [review]

File: docs/team-process.md [hunk 2]
```diff
@@ -9,29 +9,29 @@ Here are some few resources for documentation associated with onboarding:
 1. [Onboarding checklist](https://docs.google.com/document/d/1zhvp8vcWnVSvUbTKZ-0sHrf5lA6OTlpXqvjeLgKJtVk)
 2. [Onboarding Reference Guide](https://docs.google.com/document/d/1jOd0wAsqTDBKsFeEvUj9ezvdTimTsqTwkYd6Imibl3c)
 
-If you do not have permissions to view these documents, please reaach out to the scrum master. 
+If you do not have permissions to view these documents, please reaach out to the scrum master.
```
Comment (jimlerza) [inline docs/team-process.md]: I know this isn't your typo, but I just noticed:

`reaach` should be `reach`

**Review by jimlerza — APPROVED** [review]

File: docs/team-process.md (hunk 2 above)
Comment (jimlerza) [reply docs/team-process.md]: ...and now that I'm looking even closer, do we need to change "scrum master" to something else?

Comment (rgoshen) [reply docs/team-process.md]: changed...thanks Jim

**Pull request merged**